borbskeeper/Features/1TaskList/IncompleteTaskListInfiniteScrollView.m

```diff
@@ -12,6 +12,14 @@ @implementation IncompleteTaskListInfiniteScrollView
 @dynamic tableView;
 @dynamic activityIndicator;
 
+- (void)refreshTableView {
```

Comment:
We should move this logic to the infinite scroll view

Reply:
It is in the infinite scroll view I just redefined it for this infinite scroll because I needed to change the color of the indicator.